starter_scripts/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get -y update && \
     apt-get -y autoremove && \
     ldconfig
 
-RUN curl https://bootstrap.pypa.io/get-pip.py | python3
+RUN curl https://bootstrap.pypa.io/pip/3.6/get-pip.py | python3
```

Comment:
Since I think python 3.4 , python can do that for you: `python3 -m ensurepip`
It also should select the appropriate version for the interpreter.

For more details see [ensurepip](https://docs.python.org/3.6/library/ensurepip.html)

Reply:
@tkilias any concern using the `ensurepop` method?

Reply:
I created #114 for this issue.